cafebabel/articles/models.py

```diff
@@ -81,8 +81,7 @@ def save_from_request(self, request):
         if current_user.has_role('editor'):
             if not self.editor:
                 data['editor'] = current_user.id
-            if data.get('author'):
-                data['author'] = User.objects.get(id=data.get('author'))
+            data['author'] = User.objects.get(id=data['author'])
```

Comment:
Wondering about editing an article where the editor has removed his profile.
Edge case, not blocking, just sharing the wonder ;)

Reply:
That's one of the many reasons why we should *never* delete a user :)